javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/OpenTelemetryInstaller.java

```diff
@@ -79,28 +78,6 @@ static ConfigProperties getDeclarativeConfigBridgedProperties(
         .buildFromInstrumentationConfig(configProvider.getInstrumentationConfig());
   }
 
-  private static boolean defaultEnabled(ConfigProvider configProvider) {
```

Comment:
moved this method to AgentConfig so I could use it from agent instrumentation modules